Package.swift

```diff
@@ -88,8 +88,12 @@ let package = Package(
     .target(
       name: "IndexStoreDB_LLVMSupport",
       dependencies: [],
-      path: "lib/LLVMSupport"),
+      path: "lib/LLVMSupport",
+      cxxSettings: [
+        .define("LLVM_ON_WIN32", .when(platforms: [.windows])),
```

Comment:
Does this propagate to dependent targets, or is that not needed? E.g. when the other targets `#include` llvm headers from this target, will they also need `LLVM_ON_WIN32`?